### PR TITLE
feat: quick access module cards for common apis

### PR DESCRIPTION
**Pull request by janvennemann**
This turns the existing list of commonly used APIs into material design like cards which give a nice and clear overview.

### CODE REVIEW

File: docs/.vuepress/local-components/api/ModuleCard.vue [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="docs-card rounded">
+    <router-link :to="link" class="block p-6">
```
Comment (sgtcoolguy) [inline docs/.vuepress/local-components/api/ModuleCard.vue:3]:
```suggestion
    <router-link :to="link" class="block p-4">
```
Along with making the icon float right, this helps make the cards more compact.

Here's a visual diff to show:

Screen Shot 2020-10-21 at 3 32 15 PM


Screen Shot 2020-10-21 at 3 33 12 PM

File: docs/.vuepress/local-components/api/ModuleCard.vue [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  &:hover
+    box-shadow: 0 2px 4px 0 rgba(14, 30, 37, .12), 0 4px 16px 1px rgba(14, 30, 37, .06)
+
+  > a
+    color inherit
+    .content:not(.custom) &:hover
+      text-decoration none
+
+    .card-icon
+      background-color $primary-ltr
```
Comment (sgtcoolguy) [inline docs/.vuepress/local-components/api/ModuleCard.vue:43]: As per suggestion above, combined these help the cards not take up so much vertical space.
```suggestion
      background-color $primary-ltr
      float right
```

**Comment by janvennemann** [comment]
@sgtcoolguy I modified your suggestions a little and used a flex layout to properly align the card title and icon. This creates a little more space between heading and text so i also kept the padding of the cards in general as is, since it looked off/too dense with less padding around the card content than between header/icon and text. That way we still save space but keep everything properly aligned.

**Comment by sgtcoolguy** [comment]
Much better, feels less vertically "bloated". Without the right-hand sidebar, still feels a bit too spacey to me, but at least on my monitor I can fit 2 groups on the screen at once now.

**Pull request merged**